jamovi/ranova.a.yaml

```diff
@@ -92,21 +92,12 @@ options:
         - median
         - trim
         - boot
-      default:
-          trim
+      default: trim
       description:
           R: >
-            `'median'`, `'trim'` (default) or `'boot'`; the method to use,
+            `'median'`, (default) `'trim'` or `'boot'`; the method to use,
```

Comment:
we have used the convention of putting `(default)` after the value, rather than before.

i.e. https://www.rdocumentation.org/packages/jmv/versions/2.3.4/topics/descriptives

Reply:
Yes you are right.
I kept a previous .a.yml, where it defaulted to median and with trim, I only moved the comma to the left in the description.
Fixed

Reply:
if you've fixed this, you need to commit and push to your branch in order for it to show up in this PR.